<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDeviceWindow</class>
 <widget class="QMainWindow" name="RegisterDeviceWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>장비정보등록</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="registDeviceText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(94, 92, 100);
border:none;</string>
    </property>
    <property name="text">
     <string>라우터 정보 등록</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>561</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="registIPAddressText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell Light</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(94, 92, 100);
border:none;</string>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registIPAddress">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="registCommunityText">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell Light</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(94, 92, 100);
border:none;</string>
     </property>
     <property name="text">
      <string>Community</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registCommunity">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="registPushButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>등록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deletePushButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string> 삭제</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="routerListTable">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>391</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>193</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>IP 주소</string>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Community</string>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
